go.mod

```diff
@@ -238,3 +238,5 @@ require (
 	lukechampine.com/blake3 v1.2.1 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/livepeer/ai-worker => github.com/Livepool-io/ai-worker v0.0.0-20240806021536-468d65dca834
```

Comment:
@kyriediculous has to be removed when new ai-worker is released. Then the ai-worker dependency has to be updated.